### PR TITLE
Package ppx_string_interpolation.1.0.0

### PR DESCRIPTION
**Pull request by ksromanov**
### `ppx_string_interpolation.1.0.0`
String interpolation PPX preprocessor

---
* Homepage: https://https://github.com/bloomberg/ppx_string_interpolation
* Source repo: git+https://github.com/bloomberg/ppx_string_interpolation.git
* Bug tracker: https://https://github.com/bloomberg/ppx_string_interpolation/issues

### CODE REVIEW

File: packages/ppx_string_interpolation/ppx_string_interpolation.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    checksum: "md5=3e65a57fd7fb6c7025e7ade74b22c83a"
+    }
+
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+
+depends: [
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/ppx_string_interpolation/ppx_string_interpolation.1.0.0/opam]:
```suggestion
  "dune" {>= "1.5"}
```
Dune should not be tagged as a build dependency (see https://github.com/ocaml/opam-repository/pull/14266). Furthermore your `dune-project` file says this package requires at least dune 1.5

File: packages/ppx_string_interpolation/ppx_string_interpolation.1.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "ocaml" {>= "4.04.1"}
+  "ppxlib" {build}
```
Comment (kit-ty-kate) [inline packages/ppx_string_interpolation/ppx_string_interpolation.1.0.0/opam]:
```suggestion
  "ppxlib"
```
This package seems to be used in a library. Libraries need to be rebuilt when one of their dependencies change

File: packages/ppx_string_interpolation/ppx_string_interpolation.1.0.0/opam [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  "sedlex" {build}
```
Comment (kit-ty-kate) [inline packages/ppx_string_interpolation/ppx_string_interpolation.1.0.0/opam]:
```suggestion
  "sedlex"
```
This package seems to be used in a library. Libraries need to be rebuilt when one of their dependencies change

File: packages/ppx_string_interpolation/ppx_string_interpolation.1.0.0/opam (hunk 1 above)
Comment (ksromanov) [reply packages/ppx_string_interpolation/ppx_string_interpolation.1.0.0/opam]: Thanks, addressed.

File: packages/ppx_string_interpolation/ppx_string_interpolation.1.0.0/opam (hunk 2 above)
Comment (ksromanov) [reply packages/ppx_string_interpolation/ppx_string_interpolation.1.0.0/opam]: Thanks, addressed.

File: packages/ppx_string_interpolation/ppx_string_interpolation.1.0.0/opam (hunk 3 above)
Comment (ksromanov) [reply packages/ppx_string_interpolation/ppx_string_interpolation.1.0.0/opam]: Thanks, addressed.

File: packages/ppx_string_interpolation/ppx_string_interpolation.1.0.0/opam [hunk 4]
```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+name: "ppx_string_interpolation"
+synopsis: "String interpolation PPX preprocessor"
+description: "
+String interpolation PPX preprocessor.
+"
+maintainer: "Konstantin Romanov <konstantinsromanov@gmail.com>"
+authors: ["Konstantin Romanov <konstantinsromanov@gmail.com>"]
+homepage: "https://https://github.com/bloomberg/ppx_string_interpolation"
```
Comment (kkazuo) [inline packages/ppx_string_interpolation/ppx_string_interpolation.1.0.0/opam]: I'm not reviewer and not maintainer, and I'm not going to disturb the review process,
but just for your information.
Is this typo?
`https://https://github.com/...`  (https:// twice)

Comment (ksromanov) [reply packages/ppx_string_interpolation/ppx_string_interpolation.1.0.0/opam]: Thanks a lot!

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**